evap/development/management/commands/reload_testdata.py

```diff
@@ -27,4 +32,9 @@ def handle(self, *args, **options):
 
         logged_call_command(self.stdout, "refresh_results_cache")
 
+        upload_dir = settings.DATADIR / "upload"
+        if upload_dir.exists():
+            shutil.rmtree(upload_dir)
```

Comment:
Maybe add a hint that this will happen to `help` and the warning print text above (so that noone can claim we didn't tell them)

Reply:
What do you think about the wording I pushed?

Reply:
sounds good, thanks